config/locales/devise.nl.yml

```diff
@@ -0,0 +1,63 @@
+# if me: DO NOT modify this file. if you need to add translations, add them to en.yml
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
+
+nl:
+  devise:
+    confirmations:
+      confirmed: "Je emailadres is met succes bevestigd."
+      send_instructions: "Je zult een email ontvangen ter bevestiging en verdere instructies binnen enkele minuten."
+      send_paranoid_instructions: "Indien je emailadres reeds in onze database staat, zul je een email ontvangen met instructies om je account te activeren binnen enkele minuten."
+    failure:
+      already_authenticated: "Je bent reeds ingelegd."
+      inactive: "Je account is nog niet geactiveerd."
+      invalid: "Invalide %{authentication_keys} of wachtwoord."
+      locked: "Je account is vergrendeld."
+      last_attempt: "Je hebt nog één poging voordat je account wordt vergrendeld."
+      not_found_in_database: "Invalide %{authentication_keys} of wachtwoord."
+      timeout: "Je sessie is verlopen. Log nogmaals in om door te gaan."
+      unauthenticated: "Je moet inloggen of aanmelden voordat je door kunt gaan."
+      unconfirmed: "Je moet je emailadres bevestigen voordat je door kunt gaan."
+    mailer:
+      confirmation_instructions:
+        subject: "Bevestiging instructies"
+      reset_password_instructions:
+        subject: "Reset wachtwoord instructies"
+      unlock_instructions:
+        subject: "Ontgrendel instructies"
+      password_change:
+        subject: "Wachtwoord veranderd"
+    omniauth_callbacks:
+      failure: "Kan je niet authenticeren via %{kind} omdat \"%{reason}\"."
+      success: "Successvol geauthenticeerd via %{kind} account."
+    passwords:
+      no_token: "Je kan niet op deze pagina komen zonder uit een wachtwoord reset email te komen. Indien je wel uit zo'n email komt, let er op dat je de volledige URL hebt overgenomen."
+      send_instructions: "Binnen enkele minuten zul je een email ontvangen met instructies over hoe je je wachtwoord kunt resetten."
+      send_paranoid_instructions: "ALs je emailadres reeds in onze database staat, dan zul je instructies ontvangen om je wachtwoord te resetten."
+      updated: "Je wachtwoord is ingesteld. Je bent nu ingelogd."
+      updated_not_active: "Je wachtwoord is met succes aangepast."
+    registrations:
+      destroyed: "Tot ziens! Je account is met succes verwijderd. We hepen je in de nabije toekomst weer te zien."
+      signed_up: "Welkom! Je hebt je met succes aangemeld."
+      signed_up_but_inactive: "Je hebt je met succes aangemeld, maar we konden je niet inloggen omdat je account nog niet is geactiveerd."
+      signed_up_but_locked: "Je hebt je met succes aangebeld, maar we konden je niet inloggen omdat je account is vergrendeld."
```

Comment:
Change "aangebeld" to "aangemeld".

(This typo is funny to me, because "aangebeld" is for ringing a doorbell :D).